src/stacked_borrows.rs

```diff
@@ -270,6 +271,11 @@ impl<'tcx> Stack {
 
     /// Check if the given item is protected.
     fn check_protector(item: &Item, tag: Option<Tag>, global: &GlobalState) -> InterpResult<'tcx> {
+        if let Tag::Tagged(id) = item.tag {
+            if Some(id) == global.tracked_id {
+                throw_unsup!(Unsupported(format!("disabling item {:?} for tag {:?}", item, tag)));
```

Comment:
Uh, "unsup" really doesn't fit at all here...

Reply:
None of the others fit either. I chose one where I could specify a pure string without actually signalling any additional information by the name of the variant

Reply:
Please add a FIXME to use a `MachineError` variant (or so) then, once we added that.

Reply:
r=me with this comment and after squashing.

Reply:
With https://github.com/rust-lang/miri/pull/1092 we now have the `MachineStop` variant available, so could you use that instead?

Reply:
@oli-obk  This concern is still open.

Reply:
Yea I'm currently unable to build miri with the toolchain script. Getting some errors about libsyntax not found, will try again with nightly

Reply:
What's your version of `rustup-toolchain-install-master`? You might have to upgrade for it to properly support `rustc-dev`.

Reply:
That was it, thanks

Reply:
This gave me an idea wrt continuing execution, I hope you don't hate it too much :laughing: We may corrupt some state with it.